dist/util/types/connectionTypes.d.ts

```diff
@@ -0,0 +1,21 @@
+export interface IConnection {
+    federationURL: string;
+    apiKey: string;
+    agent: Function | string;
+    absolutePath?: boolean;
+}
+export interface IOptions {
+    method: string;
+    headers: headersType;
+    body: string;
+}
+export declare type headersType = {
+    Authorization: string;
+    "x-amz-target": string;
+};
+export declare type endpointsType = {
```

Comment:
one line break after each.

Reply:
@VikasPatel these are declaration files created via script do you still want me to add the line break